### ISSUE TITLE
Localization?

### ISSUE THREAD

**Issue by mthld** [mailcow/mailcow-dockerized-docs#184]
Hi,

What do you think about hosting localized versions of the documentation? I'm not sufficiently familiar with MkDocs functionalities but it would be a good improvement to serve some translated content to users. Especially the pages about [clients](https://github.com/mailcow/mailcow-dockerized-docs/blob/master/docs/client.md), every end-user is not fluent in English.

If it is possible, I would really like to help translating the pages. We can even use a all-in-one solution (like [Crowdin](https://crowdin.com/page/open-source-project-setup-request) or [Weblate](https://weblate.org/fr/hosting/)) who offer free plans to OSS.

Thanks in advance for your feedbacks!
Best,

**Comment by andryyy** [comment]
Hi, good idea. Netdata has a selector for this:

https://docs.netdata.cloud/
https://github.com/netdata/netdata/tree/master/docs/generator

**Comment by mthld** [comment]
Hi @andryyy,

Excellent, do you know how to proceed regarding this? As you are the owner of the org, the repos, etc, would you do the setup? 🙂

**Comment by andryyy** [comment]
I think @ntimo wants to work on this anytime soon. :)

**Comment by mthld** [comment]
Hello,

It’s been 6 months, any update regarding this?

**Comment by andryyy** [comment]
Who did work on it?

**Comment by mthld** [comment]
> Who did work on it?

I'm sorry I don't understand your answer, is it a rhetorical question? I opened this issue because I would be glad to contribute on translations but I fear I can't setup this on Mailcow's behalf.

**Comment by andryyy** [comment]
No, just curious if anybody tried and encountered problems.
I thought @ntimo gave it a try. Or you.

I didn't work on it.
